index.js

```diff
@@ -20,14 +20,6 @@ module.exports = function (a, b) {
 	var aLen = a.length;
 	var bLen = b.length;
 
-	if (aLen === 0) {
```

Comment:
If `aLen` is equal to `0`, the check at line 56 kicks in ending up in the same result. So yes, the result is the same. However, this is an early exit and thus you can argue that the faster we can exit and the less code we should execute, the better. But yes, this one can safely be removed.